tools/clang/tools/dxil2spv/lib/dxil2spv.cpp

```diff
@@ -515,6 +535,84 @@ void Translator::createBufferLoadInstruction(llvm::CallInst &instruction) {
   instructionMap[&instruction] = returnVar;
 }
 
+void Translator::createBufferStoreInstruction(llvm::CallInst &instruction) {
```

Comment:
Nit: similar to `Translator::createBufferLoadInstruction(llvm::CallInst &instruction)`, could you please provide an example DXIL to SPIR-V translation?

Reply:
Done